litmus-portal/graphql-server/graph/schema.graphqls

```diff
@@ -6,7 +6,7 @@ directive @authorized on FIELD_DEFINITION
 
 type Cluster {
   cluster_id: ID!
-  project_id: ID!
+  project_id: String!
```

Comment:
Why is this string? Should be consistent with other id;s

Reply:
In most places it is String....in some it was ID...so I changed it to String.
If you want we can make everything to IDs as well but we need something consistent.
@gdsoumya

Reply:
Sure you can change to either but the other id types like `cluster_id` etc. should also be converted to the same type then

Reply:
Yes, that's the problem...it's all over the place right now....I'm going with ID then in all places @gdsoumya @rajdas98